org.aegisub.Aegisub.yml

```diff
@@ -248,3 +255,62 @@ modules:
             x-checker-data:
               type: git
               version-scheme: semantic
+  - name: DependencyControl
```

Comment:
Since flathub refused to package Aegisub plugins on separate repos.

To keep everything organised, this module with its dependencies could be moved into a separate file.

Reply:
https://github.com/flathub/flathub/pull/6859